apps/web/pages/settings/organizations/platform/oauth-clients/components/OAuthClientForm.tsx

```diff
@@ -128,13 +162,24 @@ export const OAuthClientForm: FC = () => {
           />
         </div> */}
         <div className="mt-6">
-          <TextField label="Redirect uri" required={true} {...register("redirectUri")} />
+          <TextField type="url" label="Redirect uri one" required={true} {...register("redirectUriOne")} />
+        </div>
+        <div className="mt-6">
+          <TextField type="url" label="Redirect uri two" {...register("redirectUriTwo")} />
         </div>
         <div className="mt-6">
-          <h1 className="text-base font-semibold underline">Permissions</h1>
+          <TextField type="url" label="Redirect uri three" {...register("redirectUriThree")} />
+        </div>
+        <div className="mt-6">
+          <div className="flex justify-between">
+            <h1 className="text-base font-semibold underline">Permissions</h1>
+            <Button type="button" onClick={selectAllPermissions}>
+              Select all
+            </Button>
+          </div>
           <div>{permissionsCheckboxes}</div>
         </div>
-        <Button className="mt-6" type="submit" loading={isLoading}>
+        <Button className="mt-6" type="submit" loading={isPending}>
           Submit
         </Button>
       </form>
```

Comment:
Padding left and bottom of form is not equal

<img width="312" alt="Screenshot 2024-02-27 at 10 20 35" src="https://github.com/calcom/cal.com/assets/42170848/7adddd33-4a65-4a8f-939f-61b1b25ab3bf">

Reply:
I would also add more margin to the "Submit" button because otherwise it is closer to permissions that the select all permissions button, so we need to space it away to make it clearer that its a separate action.